Add C3 Arkanoid game to README (#51)
c3lang/c3-showcase: README.md

⭐ [Armadillo Madness](https://github.com/Ronin15/Armadillo_Madness_C3) - A complete 2D casual side scroller arcade style game written for my daughter about an Armadillo shooting poop at bad guys.

> ⭐ [C3 Arkanoid](https://codeberg.org/BrunoVDR/C3-Arka) - An arkanoid clone game written with C3 language with the Raylib library.

Reviewer: If you'd like, you can also word it this way:
```suggestion
⭐ [C3 Arkanoid](https://codeberg.org/BrunoVDR/C3-Arka) - An Arkanoid clone game written in C3 with Raylib.
```
